### PR TITLE
fix tag miss in docker build

### PR DESCRIPTION
**Pull request by yxxhero**
Signed-off-by: yxxhero <aiopsclub@163.com>

fix tag miss in docker build

### CODE REVIEW

**Comment by yxxhero** [comment]
@mumoshu

File: .github/workflows/images.yaml [hunk 1]
```diff
@@ -62,6 +64,7 @@ jobs:
       - name: Build / Push
         uses: docker/build-push-action@v3
         with:
+          context: .
```
Comment (mumoshu) [inline .github/workflows/images.yaml:67]: I thought this was the default value, right? 🤔

File: .github/workflows/images.yaml [hunk 2]
```diff
@@ -37,6 +37,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```
Comment (mumoshu) [inline .github/workflows/images.yaml:41]: Ah, that makes sense. I recall that checkout actioni used `fetch-depth: 1` by default and doesn't fetch tags? 🤔

Comment (yxxhero) [reply .github/workflows/images.yaml:41]: yes. I debug this with long time.

File: .github/workflows/images.yaml (hunk 1 above)
Comment (yxxhero) [reply .github/workflows/images.yaml:67]: No. It default use git content. It will ignore `.git` dir. So git can't get the tag when build docker image.

Comment (mumoshu) [reply .github/workflows/images.yaml:67]: Ahhh that's great to know! Thanks for finding it out and sharing your solution ❤️

For anyone interested, it's documented in the build-push-action doc:

https://github.com/docker/build-push-action#git-context

**Review by mumoshu — APPROVED** [review]
LGTM. Thanks a lot for fixing it ☺️

**Pull request merged**